cosmos/operators/local.py

```diff
@@ -538,13 +538,85 @@ class DbtDocsLocalOperator(DbtLocalBaseOperator):
 
     ui_color = "#8194E0"
 
-    required_files = ["index.html", "manifest.json", "graph.gpickle", "catalog.json"]
+    required_files = ["index.html", "manifest.json", "graph.gpickle", "catalog.json", "run_results.json"]
 
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(**kwargs)
         self.base_cmd = ["docs", "generate"]
 
 
+class DbtFreshnessLocalOperator(DbtLocalBaseOperator):
```

Comment:
Minor comment: what do you think about renaming `DbtFreshnessLocalOperator` to `DbtSourceFreshnessLocalOperator`?